### DISCUSSION TITLE
The extension "hediet.vscode-drawio" doesn't work

### DISCUSSION THREAD

**Discussion by SlavikCA** [gitpod-io/openvscode-server#187] [Q&A]
### Bug description

drawio diagram editor can't be displayed in the browser

Chrome Browser logs shows:

> console.ts:137 [Extension Host] Could not get liveshare API (at e.LiveshareFeature.<anonymous> (/home/workspace/.openvscode-server/extensions/hediet.vscode-drawio-1.6.2/dist/extension/index.js:2:168861))

List of installed extensions:

> ls -lh .openvscode-server/CachedExtensionVSIXs/
> total 100M
> -rw-r--r-- 1 1000 1000 4.1M Oct 25 22:12 bmewburn.vscode-intelephense-client-1.7.1
> -rw-r--r-- 1 1000 1000  81K Oct 25 22:12 ecmel.vscode-html-css-1.9.1
> -rw-r--r-- 1 1000 1000  38M Oct 25 23:30 hediet.vscode-drawio-1.6.2
> -rw-r--r-- 1 1000 1000  46M Oct 25 22:14 redhat.java-1.0.0
> -rw-r--r-- 1 1000 1000 2.6M Oct 25 22:12 vscjava.vscode-java-debug-0.35.0
> -rw-r--r-- 1 1000 1000 414K Oct 25 22:12 vscjava.vscode-java-dependency-0.18.8
> -rw-r--r-- 1 1000 1000 2.9M Oct 25 22:14 vscjava.vscode-java-pack-0.18.6
> -rw-r--r-- 1 1000 1000 2.1M Oct 25 22:14 vscjava.vscode-java-test-0.32.0
> -rw-r--r-- 1 1000 1000 4.9M Oct 25 22:12 vscjava.vscode-maven-0.21.2

the same extension is working fine in the desktop VS Code

I'm not sure, if this is openvscode-server  issue or the extension issue. Let me know, if I need to resubmit in the extension repo.

### Steps to reproduce

1. install the extension `hediet.vscode-drawio`
2. create empty file `test.drawio.svg`
3. open the file

### Workspace affected

No, The issue reproduced in openvscode-server running via Docker

**Comment by akosyakov** [comment]
This extension seems to be depend on live-share which is a proprietary MS extension. Could you reach to extension authors to make sure that it is optional. Live-share can be used with official MS products please see: https://github.com/MicrosoftDocs/live-share/issues/3802#issuecomment-746775005
